### PR TITLE
Certain properties in Loader/Validation are now definitely always arrays

### PR DESCRIPTION
**Pull request by SidRoberts**

### CODE REVIEW

File: phalcon/loader.zep [hunk 1]
```diff
@@ -28,21 +28,25 @@ use Phalcon\Events\EventsAwareInterface;
...
+ * );
  *
- * //register autoloader
+ * // Register autoloader
  * $loader->register();
  *
- * //Requiring this class will automatically include file vendor/example/adapter/Some.php
- * $adapter = Example\Adapter\Some();
+ * // Requiring this class will automatically include file vendor/example/adapter/Some.php
+ * $adapter = new \Example\Adapter\Some();
```
Comment (sergeyklay) [inline phalcon/loader.zep]: 👍

**Comment by sergeyklay** [comment]
Thanks

**Comment by sergeyklay** [comment]
@SidRoberts Could you please rebate onto 3.0.x ?

**Comment by SidRoberts** [comment]
@sergeyklay #12266

**Pull request merged**